.github/templates/macos_ci_workflow.yml.j2

```diff
@@ -123,7 +121,7 @@ jobs:
       TEST_CONFIG: ${{ matrix.config }}
       SHARD_NUMBER: ${{ matrix.shard }}
       NUM_TEST_SHARDS: ${{ matrix.num_shards }}
-      PYTORCH_IGNORE_DISABLED_ISSUES: ${{ needs.generate-test-matrix.outputs.ignore-disabled-issues }}
+      PR_BODY: ${{ github.event.pull_request.body }}
```

Comment:
Does github.event.pull_request.body return anything for push triggered PR workflows? If not, this code doesn't work  for ciflow triggered workflows, then, right?